css/style.css

```diff
@@ -9,285 +15,2161 @@ body {
+  height: 101px;
+  left: 50px;
+  top: 144px;
+  margin: 0;
 }
 
 .name {
-    font-style: normal;
-    font-weight: bold;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
+  position: absolute;
+  height: 15px;
+  left: 1525px;
+  top: 2813px;
+  width: 115px;
+}
+
+/* Cтили 2й секции ниже */
+
+.bag {
+  background: #fae8e4;
+  height: 952px;
+  margin-top: 0;
+}
+
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
+
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px;
+}
+
+.card {
+  position: absolute;
+  width: 488px;
+  height: 556px;
+  margin-left: 879px;
+  top: 1068px;
+  background: #fff;
+  min-width: 300px;
+  box-shadow: 0 14px 28px rgba(0,0,0,.25), 0 10px 10px rgba(0,0,0,.22);
+}
+
+.bag__color {
+  position: absolute;
+  top: 375px;
+  left: 55px;
+}
+
+.color-wight,
+.color-blue,
+.color-green {
+  width: 30.18px;
+  height: 30px;
+  border-radius: 50%;
+  border: none;
+}
+
+.color-pink {
+  width: 38px;
+  height: 38px;
+  border-radius: 50%;
+  background: #f29f9f;
+  border: 1px solid #f29f9f;
+}
+
+.color-pink:hover {
+  opacity: 0.5;
+}
+
+.color-wight {
+  background: #ded;
+}
+
+.color-wight:hover {
+  opacity: 0.5;
+}
+
+.color-blue {
+  background: #a8bff9;
+}
+
+.color-blue:hover {
+  opacity: 0.5;
+}
+
+.color-green {
+  background: #90bd8f;
+}
+
+.color-green:hover {
+  opacity: 0.5;
+}
+
+span {
+  position: absolute;
+  width: 125px;
+  height: 57px;
+  left: 288px;
+  top: 365px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  color: #000;
+}
+
+.button-minus,
+.button-plus {
+  border: none;
+  background: #fae8e4;
+}
+
+.button-minus:hover,
+.button-plus:hover {
+  font-weight: bold;
+  font-size: 30px;
+}
+
+.button-number,
+.bag__button-number {
+  position: absolute;
+  width: 100px;
+  height: 44px;
+  left: 55px;
+  top: 468px;
+  border: 1.5px solid #000;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.bag__button {
+  position: absolute;
+  width: 253px;
+  height: 44px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  left: 170px;
+  top: 468px;
+  background: #fff;
+}
+
+.bag__button:hover {
+  background: pink;
+}
+
+.bag__link-arrow:hover {
+  opacity: 0.5;
+}
+
+.bag__link {
+  position: absolute;
+  width: 299px;
+  height: 58px;
+  margin-left: 884px;
+  top: 1663px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000;
+}
+
+.bag__arrow {
+  position: absolute;
+  width: 241px;
+  margin-left: 1119.5px;
+  top: 1665px;
+}
+
+/* Третья секция */
+
+.total-look {
+  top: 1852px;
+  background: #ebe3d8;
+  height: 952px;
+  margin-top: 0;
+  min-width: 350px;
+}
+
+.things {
+  position: absolute;
+  left: 400px;
+  top: 2058px;
+  box-shadow: 0 14px 28px rgba(73, 59, 59, .25), 0 10px 10px rgba(0,0,0,.22);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)